MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -89,6 +98,34 @@ public class AudioFileProber
                 Fetch(item, result, cancellationToken);
             }
 
+            string ffmpeg_path = _serverConfigurationManager.GetEncodingOptions().EncoderAppPathDisplay;
```

Comment:
```suggestion
            string ffmpeg_path = _mediaEncoder.EncoderPath;
```

Reply:
i have added this with latest commit, please let me know if ok